lib/serialize.js

```diff
@@ -115,15 +116,14 @@ module.exports = function (app, defaults) {
       app: app,
       model: model,
       method: ctx.method.name,
+      meta: ctx.meta ? utils.clone(ctx.meta) : null,
```

Comment:
JsonApi supports the meta key in several places. it could be good to also add some kind of way to defining other meta information. Eg. on a per item in collection basis via a function.